docs/advanced/2d-1d-Mixed-Setup.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 2D and 1D Mixed Setup
+hide:
+  # - navigation
+  # - toc
+---
+
+## Overview
+
+WLED supports mixing 2D and 1D setup on the same unit, the expected result is that you could use 2D fixture and still chain a strip after or setup a strip and then a matrix after on the same pin. 
```

Comment:
> same unit, the expected
Make this two sentences.  "same unit.  The expected"

For the image, you are uploading it.  So set the source to either Aircookie's repo or better yet, a relative URL "/assets/images/content/2D-1D-MIX.png"

Reply:
okay